tests/functional/test_install_wheel.py

```diff
@@ -7,21 +7,37 @@
 
 from tests.lib import create_basic_wheel_for_package, skip_if_python2
 from tests.lib.path import Path
+from tests.lib.wheel import wheel_builder
 
 
-def test_install_from_future_wheel_version(script, data):
+# assert_installed expects a package subdirectory, so give it to them
+def build_wheel(name, version, **kwargs):
+    extra_files = kwargs.setdefault("extra_files", {})
+    extra_files["{}/__init__.py".format(name)] = "# example"
+    return wheel_builder(name=name, version=version, **kwargs)
+
+
+def test_install_from_future_wheel_version(script, tmpdir):
     """
     Test installing a future wheel
     """
```

Comment:
This docstring should probably include the description of futurewheel above.

Reply:
What is the motivation? The major goal of this helper is to make the code more readable so we don't need comments like that, since it is very easy for them to go out of sync or be misinterpreted. If the implementation looks too busy or it's not obvious at a glance what is happening I would rather tackle that than add a doc string.

Reply:
I was assuming the “future wheel” means a wheel containing a future metadata version, but this is no longer obvious after `futurewheel` got removed from readme. Mentioning it here would help preserve that knowledge for future readers IMO.

Reply:
+1 on adding a description here. I don't think the goal should be to eliminate the need for *all* explanatory comments.

Reply:
Updated with a more specific description.